toolkit/imageconfigs/variants/variants.md

```diff
@@ -0,0 +1,56 @@
+# Supported Variants
+
+This document provides an overview of the supported variants for Azure Linux and
+explains their primary use cases. Each variant is optimized for a specific
+environment, with configurations tailored to ensure performance, compatibility,
+and feature support in its intended deployment context.
+
+## 1. Baremetal
+
+**Variant Configuration File**: `baremetal.yaml`
+
+### Description:
+The `baremetal` variant is designed to run directly on physical hardware. It
+includes the necessary drivers and configurations for initializing hardware
+components like RAID controllers, making it ideal for environments where direct
+hardware access is required, such as enterprise servers and data centers.
+
+## 2. QEMU Guest
+
+**Variant Configuration File**: `qemu-guest.yaml`
+
+### Description:
+The `qemu-guest` variant is optimized for virtual machines running in
+**QEMU/KVM** environments. It includes specific packages that enhance guest-host
+communication and optimize performance for virtualized hardware.
+
+## 3. Hyper-V Guest
+
+**Variant Configuration File**: `hyperv-guest.yaml`
+
+### Description:
+The `hyperv-guest` variant is specifically designed for virtual machines running
+in **Hyper-V** environments, such as **Microsoft Azure**. It includes drivers
```

Comment:
Calling Azure here explicitly can be confusing as opposed to the 2 other variants below.